nemo_skills/training/data_preparation_utils/prepare_sft_data.yaml

```diff
@@ -37,7 +41,9 @@ majority_filter:
 
 filters:
   drop_multi_boxed: true
+  remove_contamindated: false
```

Comment:
why false by default? I'd say should be true by default and we just make sure it doesn't do anything if the key isn't present

Reply:
Sure, making the default true.